src/main/scala/za/co/absa/commons/config/ConfigurationImplicits.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.commons.config
+
+import org.apache.commons.configuration.Configuration
```

Comment:
Why is this util based on configuration v1 and not configuration v2? Is it required by a project? The latest release of v1 was in 2013 and there is no further development.

Reply:
Honestly I've never come across V2 in practice. V1 is used everywhere incl Hadoop and subsequently Spark.